<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>326</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Oscilloscope</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QSlider" name="slVolt">
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(222, 221, 218);
selection-background-color: rgb(192, 191, 188);</string>
          </property>
          <property name="minimum">
           <number>-6</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbPic">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="slTime">
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(255, 190, 111);
selection-background-color: rgb(192, 191, 188);</string>
          </property>
          <property name="minimum">
           <number>-10</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>V/div</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QRadioButton" name="rbV20">
               <property name="text">
                <string>2V</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbV10">
               <property name="text">
                <string>1V</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbV5">
               <property name="text">
                <string>500mV</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbV1">
               <property name="text">
                <string>100mV</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>H/div</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QRadioButton" name="rbH100">
               <property name="text">
                <string>10ms</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbH50">
               <property name="text">
                <string>5ms</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbH10">
               <property name="text">
                <string>1ms</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbH5">
               <property name="text">
                <string>500us</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbH1">
               <property name="text">
                <string>100us</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Trigger</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QDoubleSpinBox" name="sbTrigger">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="maximum">
                <double>6.000000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="gbTriggerType">
               <property name="title">
                <string>Type</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QRadioButton" name="rbNone">
                  <property name="text">
                   <string>None</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbRising">
                  <property name="text">
                   <string>Rising</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbFalling">
                  <property name="text">
                   <string>Falling</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btOpen">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btClose">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFie">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFie"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
